include/roboteam_ai/control/PosController.h

```diff
@@ -7,7 +7,8 @@
 
 #include "RobotCommand.h"
 #include <utilities/Constants.h>
-#include <control/pid.h>
+#include <include/roboteam_ai/control/controllers/PidController.h>
```

Comment:
You should never have to write include/roboteam_ai in your include's, you can remove it here (not that important but just convention)